tests/FSharp.Compiler.ComponentTests/ErrorMessages/ClassesTests.fs

```diff
@@ -849,4 +849,39 @@ type C5 = class inherit System.MulticastDelegate override x.ToString() = ""  end
             (Error 771, Line 4, Col 25, Line 4, Col 36, "The types System.ValueType, System.Enum, System.Delegate, System.MulticastDelegate and System.Array cannot be used as super types in an object expression or class")
             (Error 771, Line 5, Col 25, Line 5, Col 40, "The types System.ValueType, System.Enum, System.Delegate, System.MulticastDelegate and System.Array cannot be used as super types in an object expression or class");
             (Error 771, Line 6, Col 25, Line 6, Col 49, "The types System.ValueType, System.Enum, System.Delegate, System.MulticastDelegate and System.Array cannot be used as super types in an object expression or class")
+        ]
+        
+
+    [<Fact>]
+    let ``Types can inherit from a single concrete type`` () =
+        Fsx """
+type ClassA() = class end
+
+type Class() =
+    inherit ClassA()
+        """
+        |> typecheck
+        |> shouldSucceed
+
+    [<Fact>]
+    let ``Types cannot inherit from multiple concrete types.`` () =
+        Fsx """
+type ClassA() = class end
+
+type ClassB() = class end
+
+type ClassC() = class end
+
+type Class() =
+    inherit ClassA()
+    inherit ClassB()
+    inherit ClassC()
+        """
+        |> typecheck
+        |> shouldFail
+        |> withDiagnostics [
+            (Error 959, Line 10, Col 5, Line 10, Col 12, "Type definitions may only have one 'inherit' specification and it must be the first declaration")
```

Comment:
@abonie 

> Do we actually want both these errors? I mean 959 and 932 for each additional inherit?

I guess arguably we do not need 959 as 932 will cover the cases for having multiple `inherit`.  Should we get ride of 959 ? 

Personally I think showing the specific ranges for inherit is more clear and guides the user toward removing rather than just using the type definition.